CONTRIBUTING.md

```diff
@@ -224,7 +224,7 @@ maintainer.
 
 ## Initial setup
 
-1. Clone this repository with `git clone https://github.com/deislabs/porter.git ~/go/src/get.porter.sh/porter`. Porter relies on being in the GOPATH.
+1. Clone this repository with `git clone https://github.com/getporter/porter.git ~/go/src/get.porter.sh/porter`. Porter relies on being in the GOPATH.
```

Comment:
I think our instructions are out-of-date now and this works when out of the GOPATH too. I'll add an issue to look at our version of Go and the instructions.